RadarSDK/Radar.m

```diff
@@ -49,7 +49,9 @@ + (void)initializeWithPublishableKey:(NSString *)publishableKey {
 
     [RadarSettings setPublishableKey:publishableKey];
     [[RadarLocationManager sharedInstance] updateTrackingFromInitialize];
-    [[RadarAPIClient sharedInstance] getConfig];
+    [[RadarAPIClient sharedInstance] getConfig:^(RadarStatus status, RadarMeta *_Nullable meta) {
```

Comment:
I don't think this `status` is used anywhere. Can you remove it?

Reply:
It's part of the function signature, and since other similar functions also take a `status` argument, I'd rather keep it. But I renamed it to `unused` here to clarify its lack of purpose in this world.